lib/core-integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -114,7 +231,512 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
+                "withdrawal": "self",
+                "validity_interval": {
+                    "invalid_before": {
+                      "quantity": 0,
+                      "unit": "slot"
+                    },
+                    "invalid_hereafter": {
+                      "quantity": 500,
+                      "unit": "slot"
+                    }
+                  }
+                }|]
+
+        rTx <- request @(ApiConstructTransaction n) ctx
+            (Link.createUnsignedTransaction @'Shelley wa) Default payload
+        verify rTx
+            [ expectResponseCode HTTP.status202
+            ]
+        -- TODO: sign/submit tx
+
+    it "TRANS_NEW_VALIDITY_INTERVAL_02 - Validity interval second should be >= 0" $ \ctx -> runResourceT $ do
+
+        liftIO $ pendingWith "Accepted but should be 403 - to be fixed in ADP-985"
+
+        wa <- fixtureWallet ctx
+
+        let payload = Json [json|{
+                "withdrawal": "self",
+                "validity_interval": {
+                    "invalid_before": {
+                      "quantity": -1,
+                      "unit": "second"
+                    },
+                    "invalid_hereafter": {
+                      "quantity": -1,
+                      "unit": "second"
+                    }
+                  }
+                }|]
+
+        rTx <- request @(ApiConstructTransaction n) ctx
+            (Link.createUnsignedTransaction @'Shelley wa) Default payload
+        verify rTx
+            [ expectResponseCode HTTP.status403
+            ]
+
+    it "TRANS_NEW_VALIDITY_INTERVAL_02 - Validity interval slot should be >= 0" $ \ctx -> runResourceT $ do
+
+        liftIO $ pendingWith "Returns 400, I think it should be 403 - to be fixed in ADP-985"
+
+        wa <- fixtureWallet ctx
+
+        let payload = Json [json|{
+                "withdrawal": "self",
+                "validity_interval": {
+                    "invalid_before": {
+                      "quantity": -1,
+                      "unit": "slot"
+                    },
+                    "invalid_hereafter": {
+                      "quantity": -1,
+                      "unit": "slot"
+                    }
+                  }
+                }|]
+
+        rTx <- request @(ApiConstructTransaction n) ctx
+            (Link.createUnsignedTransaction @'Shelley wa) Default payload
+        verify rTx
+            [ expectResponseCode HTTP.status403
+            ]
+
+    it "TRANS_NEW_VALIDITY_INTERVAL_02 - Validity interval 'unspecified'" $ \ctx -> runResourceT $ do
+
+        liftIO $ pendingWith $
+          "Currently throws: \
+          \parsing ApiValidityBound object failed, \
+          \expected Object, but encountered String \
+          \- to be fixed in ADP-985"
+
+        wa <- fixtureWallet ctx
+
+        let payload = Json [json|{
+                "withdrawal": "self",
+                "validity_interval": {
+                    "invalid_before": "unspecified",
```

Comment:
It should be that omission of the interval bound, or just setting it to null means "unspecified" -- no need to use a special string.